data-raw/save_internal_data.R

```diff
@@ -35,8 +35,14 @@ df_lag_lead      <- PrepareLeads(df_lag, groupvar = idvar, timevar,
 column_subtract_1              <- paste0(policyvar, "_lead", num_fd_lead_periods)
 df_lag_lead[column_subtract_1] <- 1 - df_lag_lead[column_subtract_1]
 
-df_EventStudyFHS_example <- df_lag_lead[c("id", "t", "z", "y_base", "x_r", "z_fd",
-                                          "z_fd_lead1", "z_fd_lead2", "z_fd_lead3",
-                                          "z_fd_lag1", "z_fd_lag2", "z_lag3","z_lead3", "eta_m")]
+v_OLS_variables <- c("id", "t", "z", "y_base", "x_r", "z_fd",
+                 "z_fd_lead1", "z_fd_lead2", "z_fd_lead3",
+                 "z_fd_lag1", "z_fd_lag2", "z_lag3","z_lead3")
 
-usethis::use_data(df_EventStudyFHS_example, overwrite = TRUE, version = 3)
+v_FHS_variables <- c(v_OLS_variables, "eta_m")
+
+df_EventStudyOLS_example <- df_lag_lead[v_OLS_variables]
+df_EventStudyFHS_example <- df_lag_lead[v_FHS_variables]
+df_sample_static  <- read_dta(sprintf('%s/simulation_data_static.dta', indir))
+
+usethis::use_data(df_EventStudyOLS_example, df_EventStudyFHS_example, df_sample_static, overwrite = TRUE, version = 3, internal = TRUE)
```

Comment:
Is it necessary to keep this code? I'm not sure why we would need an internal dataset beyond `example_data.rda`

Reply:
@santiagohermo see https://github.com/JMSLab/eventstudyr/pull/37#discussion_r1181804694. I think the code is a helpful record of how we created the datasets, please let me know if you disagree!

Reply:
Agreed @nateschor, thanks for the careful explainer! Resolving..